relayer/start.go

```diff
@@ -77,5 +77,8 @@ func (r *Relayer) Start(ctx context.Context) error {
 
 	// Start the foreground process
 	r.startProcess(ctx, &locker, r.relayerConfig.KeepAliveLoopTimeout, false, r.keepAlive)
+
+	// Immediately send a keep alive to Paloma during startup
+	_ = r.keepAlive(liblog.MustEnrichContext(ctx), &locker)
```

Comment:
This is unreachable.  The code on line 79 is a blocking loop until it hits an error.